homeassistant/components/starline/account.py

```diff
@@ -185,6 +185,7 @@ def engine_attrs(device: StarlineDevice) -> dict[str, Any]:
         """Attributes for engine switch."""
         return {
             "autostart": device.car_state.get("r_start"),
```

Comment:
So the other attribute also looks like it could be its own entity, would it make sense to also deprecate that at the same time so we can just tell the end user "all engine attributes" will be deprecated instead of 1 now and 1 in the future?

Reply:
Done